<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PingChecker</class>
 <widget class="QWidget" name="PingChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PingChecker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="PingCheckEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
